modules/channel-web/src/views/lite/components/messages/renderer/FileMessage.tsx

```diff
@@ -13,7 +12,7 @@ export const FileMessage = (props: Renderer.FileMessage) => {
 
   const { url, title, storage, text } = props.file
 
-  const extension = path.extname(url)
+  const extension = new URL(url).pathname
```

Comment:
Make sure to add a try-catch since I'm pretty sure `new URL` throws if the URL is invalid.